services/service_list.go

```diff
@@ -31,5 +31,13 @@ func (c *servicesClient) List(ctx context.Context) ([]ServiceGroup, error) {
 		return nil, fmt.Errorf("performing the request: %w", err)
 	}
 
-	return response.AllOrErr()
+	services, err := response.AllOrErr()
+
+	for _, service := range services {
+		if service.Message != "" {
+			err = fmt.Errorf("%w: %s", err, service.Message)
+		}
+	}
```

Comment:
This might result in a strange error string (`the world exploded: the sun got too hotthe worldexploded: the sun got toohot: wars escalated too high`, notice repetitions and missing spaces).

We should instead collect `service.Message`s into a slice and set `err = fmt.Errorf("%w: %v", err, svcMessages)` at the end.

Reply:
yup, I felt it was bad but wanted to put it out here so someone can suggest better alternatives 😉